modules/statics/src/coins.ts

```diff
@@ -8583,7 +8585,8 @@ export const coins = CoinMap.fromCoins([
     'Zeta (ERC20)',
     18,
     '0xf091867ec603a6628ed83d274e835539d82e9cc8',
-    UnderlyingAsset.ZETAEVM
+    UnderlyingAsset.ZETAEVM,
+    ZETAEVM_FEATURES
```

Comment:
can we have the variable named as ZETA_EVM_FEATURES? its better to read

Reply:
done